File: src/lib/supabaseClient.ts
import { createClient } from '@supabase/supabase-js';

// Validated credentials from user
const SUPABASE_URL = 'https://ufawzveswbnaqvfvezbb.supabase.co';
const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InVmYXd6dmVzd2JuYXF2ZnZlemJiIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjY1NTI0MjcsImV4cCI6MjA4MjEyODQyN30.AxihQaKViHj-e-3fkuWcgKm3-H1cmon4bazka4qEmog';

export const supabase = createClient(SUPABASE_URL, SUPABASE_KEY);

File: backend/supabase_client.py

import os
from supabase import create_client, Client

# Hardcoded for now in this environment as requested/previously seen
# Prioritize Env Vars for GitHub Actions
url: str = os.environ.get("SUPABASE_URL") or "https://ufawzveswbnaqvfvezbb.supabase.co"
key: str = os.environ.get("SUPABASE_KEY") or "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InVmYXd6dmVzd2JuYXF2ZnZlemJiIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc2NjU1MjQyNywiZXhwIjoyMDgyMTI4NDI3fQ.uf7mi1zUvLD9CmbAcwIgUNwHhQXUOg9cZ9Pk67C85iQ"

supabase: Client = create_client(url, key)
